frontend/javascripts/admin/project/transfer_all_tasks_modal.tsx

```diff
@@ -134,7 +134,7 @@ class TransferAllTasksModal extends React.PureComponent<Props, State> {
         </Modal>
       );
     } else {
-      const title = `All users with open tasks of ${project.name}`;
+      const title = `All users with pending tasks for ${project.name}`;
```

Comment:
```suggestion
      const title = `All users with active tasks for ${project.name}`;
```
(this one should have been “active” all along. Seems like we were confused about what “open” means before)